subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/TransformationNode.java

```diff
@@ -245,13 +246,16 @@ public void resolveDependencies(TaskDependencyResolver dependencyResolver, Actio
 
     private abstract class ArtifactTransformationStepBuildOperation implements CallableBuildOperation<Try<TransformationSubject>> {
 
+        @UsedByScanPlugin("The string is used for filtering out artifact transform logs in Gradle Enterprise")
+        private static final String TRANSFORMING_PROGRESS_PREFIX = "Transforming ";
+
         @Override
         public final BuildOperationDescriptor.Builder description() {
             String transformerName = transformationStep.getDisplayName();
             String subjectName = describeSubject();
             String basicName = subjectName + " with " + transformerName;
             return BuildOperationDescriptor.displayName("Transform " + basicName)
-                .progressDisplayName("Transforming " + basicName)
+                .progressDisplayName(TRANSFORMING_PROGRESS_PREFIX + basicName)
```

Comment:
Would it make sense to concatenate `TRANSFORMING_PROGRESS_PREFIX` to `basicName`? I think that way we could remove one allocation.